### ISSUE TITLE
Bitwarden in Apple App Store OSX crashing.

### ISSUE THREAD

**Issue by poudenes** [bitwarden/desktop#675]
## Describe the Bug

Tried to install latest version of Bitwarden via App Store OSX. Bitwarden version 1.24.1.
I want use this version, because only the installed app from the store can install extensions in OSX.
Only problem when installed the app crash all the time and will not even open.
The stand alone version installed from the website is working, but you can't install the extentions.

## Steps To Reproduce

1. Open App Store on your Mac device with OSX
2. Search for the Bitwarden app
3. Install the app
4. try to open the app
5. crash.

## Expected Result

That the app will open so you can use it. Install also the extentions.

## Actual Result

keeps crashing. Does not work

## Environment

MacOS Version: 10.13.6 High Sierra
iMac: 27 inch, Medio 2011, 123 GB memory

**Comment by rundqvist** [comment]
I have this issue too.

Version 1.24.5 from App Store causes main app to crash (Safari plugin works)
Version 1.24.6 downloaded from website works.

**Environment**
macOS v10.13.6 High Sierra (clean, just re-installed).

**Comment by 7174n** [comment]
Have the same issue. 

As above, Version 1.24.5 from App Store causes main app to crash (Safari plugin works).
Have not tried the website version

Environment
macOS v10.13.6 High Sierra

Directly running the Bitwarden unix executable in Bitwarden/Contents/MacOS/Bitwarden returns this in Terminal, 

/Applications/Bitwarden.app/Contents/MacOS/Bitwarden ; exit;
[486:0208/200917.102390:FATAL:gpu_data_manager_impl_private.cc(894)] The display compositor is frequently crashing. Goodbye.
Trace/BPT trap: 5
logout
Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.
Deleting expired sessions...49 completed.

[Process completed]

**Comment by mshrwtnb** [comment]
Having the similar issue.

In my case, when the app launches, loading takes forever. 
Safari extension is working fine.

```
## App Ver
1.24.5 (513) from Mac App Store

## Mac OS
macOS Big Sur 11.1
```

**Comment by mho-ohm** [comment]
My problem is almost identical to Poudenes's. I downloaded the app from the App Store to get access to the Safari extension. When I try to open it, the outline shadow of an application window (with a transparent interior) appears on screen for a few seconds, very briefly fills with white, then crashes. 

MacOS Version: 10.13.6 High Sierra
iMac: 27 inch, Early 2011
Bitwarden 1.24.5

One additional note. I did successfully install Bitwarden from the DMG file I downloaded directly from Bitwarden.com. I deleted the app when I found out that it wasn't able to install the Safari extension and attempted to use the App Store version.

**Comment by jslawler** [comment]
See attached for app crash logs.

macOS High Sierra Version 10.13.6 (17G14033)
App Store Version 2.4 (658.2)
Bitwarden Version 1.25.1

[Archive.zip](https://github.com/bitwarden/desktop/files/6240576/Archive.zip)

**Comment by cscharf** [comment]
Hi @poudenes , our QA team was able to reproduce this issue, however at the moment we have another issue open with Apple waiting on a fix, in the meantime we have a frozen build [v1.24.7](https://github.com/bitwarden/desktop/releases/tag/v1.24.7) that should allow you continue using Bitwarden with the prior App Extension for Safari. This build will not ask you to update itself. In the meantime I am closing this issue. Please let us know if that build doesn't work for you.

**Comment by poudenes** [comment]
Hi, the build is working for me...  Enabled the safari extension also.
